index.html

```diff
@@ -189,7 +196,21 @@ <h3>Skills</h3>
                 </div>
             </div>
         </section>
-    </main>
+    </section>
+
+    <section class="contact-section">
+        <h1 id="contact-card-title">Contact</h1>
+
+        <p>If you have an application you are interested in developing, a feature that you need
+            built or a project that needs coding. I’d love to help with it </p>
+
+        <form action="#" method="POST" class="contact-form"></form>
```

Comment:
- [ ] Please ensure that you add the formspree link as the value to the action attribute. The link should be similar to: `https://formspree.io/f/{form_id}`.
- [ ]  Ensure that your input tags are wrapped inside the form tags. The `</form>` closing tag should be after the button code(after line 212 and before line 213).